crm_claim_code/__init__.py

```diff
@@ -15,11 +16,12 @@ def create_code_equal_to_id(cr):
 
 
 def assign_old_sequences(cr, registry):
-    claim_obj = registry['crm.claim']
-    sequence_obj = registry['ir.sequence']
-    claim_ids = claim_obj.search(cr, SUPERUSER_ID, [], order="id")
+    env = Environment(cr, SUPERUSER_ID, {})
```

Comment:
Yes I think we are supposed to do it.
IIRC, if we are already inside a `with api.Environment.manage()` which is the case most of the time (we are when we come from a rpc request), it has no effect, but it is safer to use ` api.Environment.manage()` to handle cases where we would not already be in a `manage()` context.